oarepo_ui/templates/components/datafields/IValueList.jinja

```diff
@@ -1,4 +1,5 @@
 {# def #}
-<dl>
+
+<dl class="ui cf-value-list">
```

Comment:
would it be possible to name the component without the cf- prefix? The reason is that the value list can be used outside the custom fields as well (and is used in mbdb)